src/shared/header/Header.tsx

```diff
@@ -58,8 +58,8 @@ const Header = () => {
 						</div>
 						<div className="col-5 wr-header">
 							<span><NavLink to="/" exact activeClassName="active">HOME</NavLink></span>
-							<span><NavLink to="/about" activeClassName="active">SHOP</NavLink></span>
-							<span><NavLink to="/about" activeClassName="active">MY ACCOUNT</NavLink></span>
+							<span><NavLink to="/">SHOP</NavLink></span>
+							<span><NavLink to="/">MY ACCOUNT</NavLink></span>
```

Comment:
Disable or hide these until they have somewhere to lead

Reply:
I actually deleted them, but forgot that I did it on the bidding branch hah